backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/appsearchmodal/BpkAppSearchModal.kt

```diff
@@ -70,8 +72,7 @@ data class BpkAction(val text: String, val onActionSelected: () -> Unit)
 @Composable
 fun BpkAppSearchModal(
     title: String,
-    inputText: String,
-    inputHint: String,
+    inputSummary: SearchInputSummary,
```

Comment:
rather than making the breaking change please add a `prefix` parameter (like the one in the `SearchInputSummary`) and set the default to `icon`